test/sync/test_default-dataHandlers.js

```diff
@@ -0,0 +1,178 @@
+var assert = require('assert');
+var sinon = require('sinon');
+var defaultDataHandlersModule = require('../../lib/sync/default-dataHandlers.js');
+
+var id = 'datahandler_test';
+var queryParams = {};
+var metaData = {};
+
+// stubs when there is no real MongoDB
+var collectionStub = {
+  insertOne: sinon.stub().callsArgWith(1, null, {
```

Comment:
`callsArgWith` is ok. You can also use sinon.yields, which will automatically pass the give parameters to callback function. A bit easier to use.